recipes/__main__.py

```diff
@@ -0,0 +1,40 @@
+import sys
+from builtins import input
+
+from recipes.lib.get_opts import parser
+from recipes.lib.new_json import create_json
+from recipes.lib.edit_json import edit_json
+from recipes.lib.delete_json import delete_json
+
+
+def main():
+    """This function launches the retriever-recipes."""
+    if len(sys.argv) == 1:
+        # If no command line args are passed, show the help options
+        parser.parse_args(['-h'])
+    else:
+        # Otherwise, parse them
+        args = parser.parse_args()
+
+        if args.command == 'new_json':
+            # Create new JSON script
+            create_json()
+            return
+
+        if args.command == 'edit_json':
```

Comment:
I'm having a problem with edit_json where if I edit a script and change its name, it outputs a new script with the new name, but keeps the old script with the old script. I think this is not the intended effect if we "edit" a script.

Reply:
@zhangcandrew this is the exact behavior as seen in `retriever`, however, we can change it if desired.

Reply:
Yeah I think I remember talking about this with @henrykironde before. We can leave it this way for now then, but it might be worth it to add a note in the docs somewhere in case someone uses this functionality in the future and gets confused.

Reply:
Created an issue for this.